config/coverage.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
   coverageEnvVar: 'COVERAGE',
   coverageFolder: 'coverage',
+  excludes: [
+    /app\/(.*)/,
+    /dummy\/(.*)/
```

Comment:
I actually don't want to exclude dummy as eventually I plan to add acceptance tests that ensure the dummy app continues to work properly.